cruise.umple/src/generators/Generator_CodePlainRequirementsDoc.ump

```diff
@@ -27,7 +30,30 @@ class PlainRequirementsDocGenerator
   void generate(){
     doc.createParagraphElement("Plain Requirements Doc from "+model.getUmpleFile().getSimpleFileName()+".ump", false);
     UmpleModelWalker walker = new UmpleModelWalker(model);
-    walker.accept(new ReqVisitor(doc));
+    //System.out.println(model.getAllRequirements().values());
+    TreeSet<Requirement> ReqTreeSet;
+    if (model.modelHasSuboption("PlainRequirementsDoc", "reqSortID")) {
+      Comparator<Requirement> IdComparator = (r1, r2) -> r1.getIdentifier().compareTo(r2.getIdentifier());
+      // Initialize with ID Comparator
+      ReqTreeSet = new TreeSet<>(IdComparator); 
+    } 
+    else if (model.modelHasSuboption("PlainRequirementsDoc", "reqSortStat")) {
+      Comparator<Requirement> StatComparator = (r1, r2) -> r1.getStatement().compareTo(r2.getStatement());
+      // Initialize with Statement Comparator
+      ReqTreeSet = new TreeSet<>(StatComparator); 
+    } 
+    else {
```

Comment:
This block is setting the comparator the same as the if block on line 36. So there should just be an if as on line 36 and then the else on line 40, or else